File: wa/wa_Assistant_DevOps/updateWorkspace.pl
#!/usr/bin/perl -w


$toUserName = "xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx";
$toPassword = "yyyyyyyyyyyy";
$answerFile = "./path_to/answers_added.json";
$toWorkspace = "zzzzzzzz-zzzz-zzzz-zzzz-zzzzzzzzzzzz";

$api_url = "https://gateway.watsonplatform.net/assistant/api";

`curl -k -u "$toUserName":"$toPassword" -X POST --header "Content-Type: application/json" --data @"$answerFile" "$api_url/v1/workspaces/$toWorkspace?version=2018-02-16"`;

File: wa/wa_Assistant_DevOps/updateWorkspace.py
import os

toUserNaem="XXXX-XXXX-XXXX-XXXX"
toPassword = "YYYYYY"
answerFile = './path_to/answers_added.json"
toWorkspace = "zzzzzz-zzzzz-zzzz"

api_url="https://gateway.watsonplatform.net/assistant/api"

os.system('curl -k -u"' + toUserNeame + '":"' + toPassword+ '" -X POST --header "Content-Type: application/json" --data @"' + answerFile + '" "' + apu_url + '/v1/workspaces/' + toWorkspace + '?version=2018-02-16"')

